src/plopp/backends/matplotlib/line.py

```diff
@@ -229,6 +233,10 @@ def update(self, new_values: sc.DataArray):
                 )
             )
 
+    def _update_limits(self):
```

Comment:
It is a bit unfortunate that we have to remember to update the limits every time the data changes. If we add later another function that modifies the data, we may forget to also call this.